File: TransCoder_tokenized_test_set_functions/java/CHECK_WHETHER_SECOND_STRING_CAN_FORMED_FIRST_STRING_USING_COUNT_ARRAY.java
static boolean canMakeStr2 ( String str1, String str2 ) {
  int [ ] count = new int [ MAX ];
  char [ ] str3 = str1 . toCharArray ( );
  for ( int i = 0;
  i < str3 . length;
  i ++ ) count [ str3 [ i ] ] ++;
  char [ ] str4 = str2 . toCharArray ( );
  for ( int i = 0;
  i < str4 . length;
  i ++ ) {
    if ( count [ str4 [ i ] ] == 0 ) return false;
    count [ str4 [ i ] ] --;
  }
  return true;
}

File: TransCoder_tokenized_test_set_functions/cpp/CHECK_WHETHER_SECOND_STRING_CAN_FORMED_FIRST_STRING_USING_COUNT_ARRAY.cpp
bool canMakeStr2 ( string str1, string str2 ) {
  int count [ MAX ] = {
    0 };
    for ( int i = 0;
    i < str1 . length ( );
    i ++ ) count [ str1 [ i ] ] ++;
    for ( int i = 0;
    i < str2 . length ( );
    i ++ ) {
      if ( count [ str2 [ i ] ] == 0 ) return false;
      count [ str2 [ i ] ] --;
    }
    return true;
  }